README.md

```diff
@@ -12,8 +12,10 @@ See the [RELEASES](./RELEASES.md) list.
 
 ### Legacy
 
-The legacy command-line version of this code is represented by the `v3-legacy` branch. That branch is deployed to `sul-lyberservices-test`
-and `-prod`, and is still actively used by a small number of power users in the PSM group. Until the [desired functionality](https://github.com/sul-dlss/pre-assembly/issues/221) in the `v3-legacy` branch has been ported to the web application in `master`, we should continue to maintain the `v3-legacy` branch.
+The legacy command-line version of this code is represented by the `v3-legacy` branch. That branch in -stage is deployed to `sul-lyberservices-test`
+and in -prod is deployed to `sul-lyberservices-prod` and is still actively used by a small number of power users in the PSM group. Until the
+[desired functionality](https://github.com/sul-dlss/pre-assembly/issues/221) in the `v3-legacy` branch has been ported to the web application in `master`,
+we should continue to maintain the `v3-legacy` branch.
```

Comment:
why?  until when?

Reply:
That discussion is enumerated in the linked github issue: https://github.com/sul-dlss/pre-assembly/issues/221